README.md

```diff
@@ -1149,6 +1155,9 @@ IEEE Robotics and Automation Letters (RA-L) 2024. [PDF](https://arxiv.org/pdf/24
 *[Deep Polarization Reconstruction with PDAVIS Events](https://openaccess.thecvf.com/content/CVPR2023/html/Mei_Deep_Polarization_Reconstruction_With_PDAVIS_Events_CVPR_2023_paper.html)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://openaccess.thecvf.com/content/CVPR2023/papers/Mei_Deep_Polarization_Reconstruction_With_PDAVIS_Events_CVPR_2023_paper.pdf), [code](https://github.com/SensorsINI/e2p).
 
+- <a name="Muglikar23cvpr"></a>Muglikar, M., Bauersfeld, L., Moeys, D., Scaramuzza, D.,  
+*[Event-based Shape from Polarization](https://arxiv.org/abs/2301.06855)*,  
```

Comment:
the official link should be https://openaccess.thecvf.com/content/CVPR2023/papers/Muglikar_Event-Based_Shape_From_Polarization_CVPR_2023_paper.pdf  correct?

Reply:
yes, updated the PDF to this link and doi